pkg/services/sqlstore/migrations/ualert/channel.go

```diff
@@ -0,0 +1,152 @@
+package ualert
+
+import (
+	"fmt"
+
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/pkg/labels"
+
+	"github.com/grafana/grafana/pkg/components/securejsondata"
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	apimodels "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
```

Comment:
The biggest thing that needs to change is the imports. Since this is a point in time migration against the database, once we enable it (remove the env variable) than we shouldn't really ever change the code.

So things like prometheus and apimodels etc should not be imported, rather a snapshot of the code needed and copied into here.

Reply:
https://github.com/grafana/grafana/pull/33846/commits/ce7750fbefee1ced669d1cf8ae227b10cd6068c8 should take care of it